src/Agent.Plugins/PipelineCache/PipelineCachePlugin.cs

```diff
@@ -0,0 +1,66 @@
+using System;
```

Comment:
Can you please change the order to match DedupManifestArtifactClientFactory.cs once - or vice-versa , for consistency.

Reply:
I prefer to have System at top

---
In reply to: [274598722](https://github.com/Microsoft/azure-pipelines-agent/pull/2184#discussion_r274598722) [](ancestors = 274598722)